package.json

```diff
@@ -34,7 +34,8 @@
     "grunt-symlink": "^0.4.0",
     "grunt-verbosity": "^1.0.1",
     "jquery": "^3.3.1",
-    "kss": "~3.0.0-beta.13",
-    "postcss-cli": "^5.0.0"
+    "kss": "~3.0.0-beta.22",
+    "postcss-cli": "^5.0.0",
+    "breakpoint-sass": "^2.7.1"
```

Comment:
By 'this' you mean `breakpoint-sass`, right?

Reply:
I can confirm that is what he mean by "this." Because we're no longer using it, there should be no issue with removing it from the package. Richard and I discussed this earlier today.